catalog.html

```diff
@@ -79,11 +79,23 @@
                 </ul>
               </fieldset>
             </li>
-            <li>
+            <li class="search-parameter right">
               <!-- Выбор стоимости в сутки  -->
               <fieldset class="search-parameter">
                 <legend class="infrastructure-item">Стоимость в сутки (Р):</legend>
-                <!-- Здесь будет форма  -->
+                <div>
+                  <input type="text" name="min-price" id="min-price" placeholder="От 0">
+                  <label for="min-price">Минимальная стоимость</label>
+
+                  <input type="text" name="mах-price" id="mах-price" placeholder="до 3000">
+                  <label for="mах-price">Максимальная стоимость</label>
```

Comment:
```suggestion
                  <label for="mах-price">До</label>
```
И поменять местами `label` с `input`.